### PR TITLE
Informative 2

### PR DESCRIPTION
**Pull request by LysandreJik**

### CODE REVIEW

File: src/transformers/models/marian/convert_marian_to_pytorch.py [hunk 1]
```diff
@@ -701,7 +701,12 @@ def unzip(zip_path: str, dest_dir: str) -> None:
     """
     parser = argparse.ArgumentParser()
     # Required parameters
-    parser.add_argument("--src", type=str, help="path to marian model sub dir", default="en-de")
+    parser.add_argument(
+        "--src",
+        type=str,
+        help="path to marian model sub dir. yaml.load will be used to load the configuration file, please be weary of which file you're loading.",
```
Comment (lewtun) [inline src/transformers/models/marian/convert_marian_to_pytorch.py:707]: turbo nit (although i realise these reports are tiresome :))
```suggestion
        help="path to marian model sub dir. yaml.load will be used to load the configuration file, please be wary of which file you're loading.",
```

File: src/transformers/models/maskformer/convert_maskformer_original_pytorch_checkpoint_to_pytorch.py [hunk 2]
```diff
@@ -641,7 +641,8 @@ def get_name(checkpoint_file: Path):
         type=Path,
         help=(
             "A directory containing the model's checkpoints. The directory has to have the following structure:"
-            " <DIR_NAME>/<DATASET_NAME>/<CONFIG_NAME>.pkl"
+            " <DIR_NAME>/<DATASET_NAME>/<CONFIG_NAME>.pkl\n"
+            "Given the files are in the pickle format, please be weary of passing it files you trust."
```
Comment (lewtun) [inline src/transformers/models/maskformer/convert_maskformer_original_pytorch_checkpoint_to_pytorch.py:645]:
```suggestion
            "Given the files are in the pickle format, please be wary of passing it files you trust."
```

File: src/transformers/models/mobilevitv2/convert_mlcvnets_to_pytorch.py [hunk 3]
```diff
@@ -314,7 +314,12 @@ def convert_mobilevitv2_checkpoint(task_name, checkpoint_path, orig_config_path,
     parser.add_argument(
         "--orig_checkpoint_path", required=True, type=str, help="Path to the original state dict (.pt file)."
     )
-    parser.add_argument("--orig_config_path", required=True, type=str, help="Path to the original config file.")
+    parser.add_argument(
+        "--orig_config_path",
+        required=True,
+        type=str,
+        help="Path to the original config file. yaml.load will be used to load the file, please be weary of which file you're loading.",
```
Comment (lewtun) [inline src/transformers/models/mobilevitv2/convert_mlcvnets_to_pytorch.py:321]:
```suggestion
        help="Path to the original config file. yaml.load will be used to load the file, please be wary of which file you're loading.",
```

File: src/transformers/models/reformer/convert_reformer_trax_checkpoint_to_pytorch.py [hunk 4]
```diff
@@ -202,7 +202,12 @@ def convert_trax_checkpoint_to_pytorch(trax_model_pkl_path, config_file, pytorch
     parser = argparse.ArgumentParser()
     # Required parameters
     parser.add_argument(
-        "--trax_model_pkl_path", default=None, type=str, required=True, help="Path to the TensorFlow checkpoint path."
+        "--trax_model_pkl_path",
+        default=None,
+        type=str,
+        required=True,
+        help="Path to the TensorFlow checkpoint path.\n"
+        "Given the files are in the pickle format, please be weary of passing it files you trust.",
```
Comment (lewtun) [inline src/transformers/models/reformer/convert_reformer_trax_checkpoint_to_pytorch.py:210]:
```suggestion
        "Given the files are in the pickle format, please be wary of passing it files you trust.",
```

**Comment by LysandreJik** [comment]
I appreciate the review @lewtun, thank you :grin:

**Pull request merged**